### ISSUE TITLE
remove `contributor` and `author` fields

### ISSUE THREAD

**Issue by WindowsMeosu** [WindowsMeosu/peppypedia#114]
reason: should contributor fields expand (unlikely, considering the state of this project and other factors, but just in case), then the article pretty much becomes displeasing to look at. If that doesn't sound like a big deal to you, do keep in mind that these fields cannot be aligned from a right or left position and are instead forced below whatever the fuck is above them (in this case, another contributor field). As for authors, I'd rather credit them as contributors instead. While I do want to acknowledge contributors, I don't want to make the article look utterly more displeasing than 4chan, and that is not just counting visually.

**Comment by WindowsMeosu** [comment]
TLDR: removing because at this point it's completely pointless like random @everyone mentions on discord or whatever
